src/spec.md

```diff
@@ -73,11 +82,14 @@ Each `update` which is passed to the callback comes with the following propertie
 
 - `update.info`: optional, short, informational message (see [`sendUpdate()`](#sendupdate))
 
-- `update.document`: optional, document name as set by the sender, (see [`sendUpdate()`](#sendupdate)),
-  implementations show the document name e.g. beside the app icon or in the title bar
+- `update.document`: optional, document name as set by the sender, (see [`sendUpdate()`](#sendupdate)).
+  Implementations show the document name e.g. beside the app icon or in the title bar.
+  There is no need for the receiver to further process this information.
 
 - `update.summary`: optional, short text, shown beside icon (see [`sendUpdate()`](#sendupdate))
 
+Calling `setUpdateListener()` multiple times on the same webxdc object is undefined behavior.
```

Comment:
> on the same webxdc object

this part is confusing, there is only one of those and it is not called "object" by us at prior points.
I think you mean multiple times in one page or app?

Reply:
it is only a sidenote, to answer corner case questions shortly :) i think, it is clear that webxdc is an object - and yes you may have multiple on different pages.

please do a concrete wording suggestion or commit to improve

Reply:
I bet this is not clear to noobs.